cmd/crates/soroban-spec-tools/src/lib.rs

```diff
@@ -1147,6 +1158,7 @@ impl Spec {
                     ScSpecEntry::UdtUnionV0(union) => self.arg_value_union(union, depth),
                     ScSpecEntry::UdtEnumV0(enum_) => Some(arg_value_enum(enum_)),
                     ScSpecEntry::FunctionV0(_) | ScSpecEntry::UdtErrorEnumV0(_) => None,
+                    ScSpecEntry::EventV0(_) => todo!("EventV0 is not supported yet"),
```

Comment:
Should be `None` like the line above, because an event type cannot be used as a user defined type.